### PR TITLE
add dispatch CLI to the OVA

### PR DESCRIPTION
**Pull request by kars7e**
* Add Dispatch CLI and its config to the OVA
* open port 443 for API Gateway
* set the api-endpoint to docker0 ip address for event drivers

### CODE REVIEW

File: ova/scripts/systemd/units/dispatch-server.service [hunk 1]
```diff
@@ -11,7 +11,7 @@ Restart=on-failure
 RestartSec=10
 
 WorkingDirectory=/var/lib/dispatch
-ExecStart=/usr/bin/dispatch-server --host 0.0.0.0 --enable-tls --tls-certificate /storage/data/certs/server.crt --tls-certificate-key /storage/data/certs/server.key
+ExecStart=/usr/bin/dispatch-server --api-endpoint 172.17.0.1:8080 --host 0.0.0.0 --gateway-tls-port 443 --enable-tls --tls-certificate /storage/data/certs/server.crt --tls-certificate-key /storage/data/certs/server.key
```
Comment (berndtj) [inline ova/scripts/systemd/units/dispatch-server.service]: Is that api-endpoint right?  Will it always be addressible at that IP?

Comment (tenczar) [reply ova/scripts/systemd/units/dispatch-server.service]: At the very least I think it should be given a descriptive variable name so people know what that IP is supposed to be.

Comment (kars7e) [reply ova/scripts/systemd/units/dispatch-server.service]: @berndtj yes, this is the IP of docker0 interface, always set to this IP in photon OVA, reachable from all docker containers. 
@tenczar can you propose some name? I'm honestly at loss here. this is the endpoint that is reachable by event drivers (running as containers).

Comment (tenczar) [reply ova/scripts/systemd/units/dispatch-server.service]: Could we call it DOCKER_0_BRIDGE_IP or something similar?

Comment (kars7e) [reply ova/scripts/systemd/units/dispatch-server.service]: Renamed to events-gateway. Thanks @berndtj !

**Pull request merged**